project.owasp.yaml

```diff
@@ -0,0 +1,110 @@
+audience:
+  - breaker
+  - builder
+community:
+  - name: defectdojo
+    platform: slack
+    url: https://owasp.slack.com/archives/C2P5BA8MN
+    description: DefectDojo Project Discussion
+  - name: defectdojo-dev
+    platform: slack
+    url: https://owasp.slack.com/archives/C014H3ZV9U6
+    description: DefectDojo development. This channel is for assistance with PRs to Defectdojo or dev questions. Please use <#C2P5BA8MN>  for general discussions.
+  - name: defectdojo-test
+    platform: slack
+    url: https://owasp.slack.com/archives/C6AV5TUAF
+  - name: defectdojo-uiv2
+    platform: slack
+    url: https://owasp.slack.com/archives/C01S34D42T1
+leaders:
+  - name: Aaron Weaver
+    email: aaron.weaver@owasp.org
+    github: aaronweaver
+  - name: Matt Tesauro
+    email: matt.tesauro@owasp.org
+    github: mtesauro
+  - name: Greg Anderson
+    email: greg.anderson@owasp.org
+    github: devGregA
+level: 4
+license:
+  - BSD-3-Clause
+  - GPL-3.0
+  - LGPL-3.0
+  - MIT
+name: OWASP Defectdojo
+pitch: The leading open source application vulnerability management tool built for DevOps and continuous security integration.
+repositories:
```

Comment:
While this looks like a simple listing of the repos under the DefectDojo GH org, some of these are archived or not used at all.

If someone is planning to do something with this list beyond making a list, it might make sense to provide a way to filter out the less important / inactive ones.